### PR TITLE
Poh Yu Quan HW1

### PR DESCRIPTION
**Pull request by pohyuquan**
How I got my changes onto GitHub:
First I forked the HW repository, next I **cloned** the forked repository onto my computer using RStudio. I created a new project in RStudio using version control (Git) and pasted the URL for my forked repository. I modified the *README.md* file in RStudio using Markdown Syntax as indicated by Github (https://guides.github.com/features/mastering-markdown/). I committed my changes and added a message "modified README" to indicate my changes. I pulled from GitHub using the "Git" tab in RStudio to update my local repository. I pushed my changes to GitHub through RStudio in the same "Git" tab as before. Finally I made a pull request to merge the forked branch with the master. 

Reflections:
The process of cloning a repository and making local changes before committing it and pushing it was fairly straight forward as I referenced the steps indicated through the CFSS "General Homework Guidelines" (http://cfss.uchicago.edu/hw00_homework_guidelines.html#homework_workflow). Although I had previously used a different command interface (cygwin), it took some getting used to in order to learn how to perform the similar commands on RStudio. I was able to Google how to preview R Markdown documents in HTML on RStudio which allowed me to preview my HTML without the need to commit and push my changes onto GitHub. There are clearly other formatting syntax that I will need to learn in order to make my Markdown document more aesthetically pleasing but overall the process of making changes and pushing and pulling onto GitHub was fairly simple and without much problems.

### CODE REVIEW

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Poh Yu Quan

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:

- Nice attempt of adding a .gitignore. There is actually a default one for R files that you can get when creating a new repo. You tried to ignore the .Rproj, but committed it anyway. Two things:

1. Commit the .gitignore before other commits
2. Pay attention to the backslash: `.Rproj.user` vs. 
```# RStudio files
.Rproj.user/
```
3. If you are using bash, you can run the command `git status` to check what files you have added before committing them. This is best practice, especially at first.

Full R .gitignore file
```
# History files
.Rhistory
.Rapp.history

# Session Data files
.RData

# Example code in package build process
*-Ex.R

# Output files from R CMD build
/*.tar.gz

# Output files from R CMD check
/*.Rcheck/

# RStudio files
.Rproj.user/

# produced vignettes
vignettes/*.html
vignettes/*.pdf

# OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
.httr-oauth

# knitr and R markdown default cache directories
/*_cache/
/cache/

# Temporary files created by R markdown
*.utf8.md
*.knit.md
```

- Otherwise, you effectively used five of the five style elements, italics, headers, list, links, and photo. Good job.
- Try to make different and more descriptive (but short) git commit messages (versus `modified README` four times). 
-  Proved you can do the workflow.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
-# hw01
-Repository for first CFSS homework assignment
+#Computing for Social Sciences, Homework 01
+
+My name is **Poh Yu Quan**. My *last name* is **Poh** and my first name is **Yu Quan**. I am a **MAPSS student** at the University of Chicago. I come from **Singapore** but I obtained my BSc at **Imperial College London** in *Biomedical Sciences*. I am concentrating in sociology and focusing on criminology.
```
Comment (Rjschwa) [inline README.md:3]: Good use of  #emphasis x 2

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+My research interests include:
+
+1. Criminology
```
Comment (Rjschwa) [inline README.md:7]: - [x] List

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+3. Hiking
+4. Travelling
+5. Programming
+  * Python
+  * R
+  * SQL
+
+Here is a handsome picture of me for your viewing pleasure. Please **don't photoshop** it. :smiley:
+
+![My Handsome Face](http://i1174.photobucket.com/albums/r603/Poh_Yu_Quan/14717174_10153835225981746_9155896163653390504_n_zpsgckh02iw.jpg)
```
Comment (Rjschwa) [inline README.md:25]: - [x] Picture

File: README.md [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+2. Next I **cloned** the forked repository onto my computer using RStudio.
+  * Created a new project in RStudio using version control (Git) and pasted the URL for my forked repository.
+3. I modified the *README.md* file in RStudio using Markdown Syntax as indicated by Github (found [here](https://guides.github.com/features/mastering-markdown/)).
+4. I committed my changes and added a message "modified README" to indicate my changes.
+5. I pulled from GitHub using the "Git" tab in RStudio to update my local repository.
+6. I pushed my changes to GitHub through RStudio in the same "Git" tab as before.
+7. I made a pull request to merge the branch with the master.
+
+##Reflections
+The process of cloning a repository and making local changes before committing it and pushing it was fairly straight forward as I referenced the steps indicated through the CFSS "General Homework Guidelines" page (found [here](http://cfss.uchicago.edu/hw00_homework_guidelines.html#homework_workflow)). Although I had previously used a different command interface (*cygwin*), it took some getting used to in order to learn how to perform the similar commands on RStudio.
```
Comment (Rjschwa) [inline README.md:41]: Evaluation HW-1 for Poh Yu Quan
----------------------------------------------------
| Topic                                 | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**                   |    x       |              |            |
| **Coding strategy**             |     x      |              |            |
| **Presentation**                  |    x        |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**               |     x      |              |            |

Remarks:

* Overall a good use of all of the required Markdown code. It was well integrated into your biography.
* I like the use of the picture, it added a lot to the visual diversity of your page?
* I would suggest adding a bit more to your reflections and commits. I think detailing out some more of your decision making process would be helpful in understanding and replication.

**Review by Rjschwa — COMMENTED** [review]
Schwarzhaupt Peer Review

**Pull request closed without merging**